control-plane/pkg/reconciler/trigger/trigger_test.go

```diff
@@ -123,6 +125,180 @@ func TestTriggerWithNewFiltersReconciler(t *testing.T) {
 	}
 }
 
+func TestTriggerReconcilerWithOIDC(t *testing.T) {
+	eventing.RegisterAlternateBrokerConditionSet(base.IngressConditionSet)
+
+	t.Parallel()
+
+	for _, f := range Formats {
+		triggerReconciliationWithOIDC(t, f, *DefaultEnv)
+	}
+}
+
+func triggerReconciliationWithOIDC(t *testing.T, format string, env config.Env) {
```

Comment:
I think we can integrate them into the `triggerReconcilation()` function.
You can use the following to set the feature flags for a test:
```
Name: "OIDC: ...",
Ctx: feature.ToContext(context.Background(), feature.Flags{
	feature.OIDCAuthentication: feature.Enabled,
}),
Objects: ...
```

That way we align more with the existing

Reply:
> I think we can integrate them into the `triggerReconcilation()` function. You can use the following to set the feature flags for a test:
> 
> ```
> Name: "OIDC: ...",
> Ctx: feature.ToContext(context.Background(), feature.Flags{
> 	feature.OIDCAuthentication: feature.Enabled,
> }),
> Objects: ...
> ```
> 
> That way we align more with the existing

I have tried that, but it didnt work .

Reply:
@xiangpingjiang: https://github.com/knative-extensions/eventing-kafka-broker/pull/3612 merged. When you rebase, you should be able to use it in `triggerReconcilation()` with a configured `Ctx`.